revzilla/braintree-elixir
# Fix typespecs and config

Fixed some typespecs, changed config to use non-deprecated stuff

QA:
Create `config/dev.secret.exs` and `config/test.secret.exs` (both files may be empty).
Run `mix compile` and `mix dialyzer`, confirm it works fine.
